### PR TITLE
Naming of message args

### PR DESCRIPTION
**Pull request by mattias-p**
This is another step towards solving #60 and #713.

### CODE REVIEW

File: lib/Zonemaster/Engine/Test/DNSSEC.pm [hunk 1]
```diff
@@ -786,7 +799,8 @@ Readonly my %TAG_DESCRIPTIONS => (
     },
     NSEC3_SIG_VERIFY_ERROR => sub {
         __x    # DNSSEC:NSEC3_SIG_VERIFY_ERROR
-          'Trying to verify NSEC3 RRset with RRSIG {sig} gave error \'{error}\'.', @_;
+'Trying to verify NSEC3 RRset with RRSIG {keytag} gave error \'{error}\'.',
+          @_;
```
Comment (matsduf) [inline lib/Zonemaster/Engine/Test/DNSSEC.pm:803]: Indentation looks wrong.

File: lib/Zonemaster/Engine/Test/DNSSEC.pm [hunk 2]
```diff
@@ -798,23 +812,25 @@ Readonly my %TAG_DESCRIPTIONS => (
     },
     NSEC_SIG_VERIFY_ERROR => sub {
         __x    # DNSSEC:NSEC_SIG_VERIFY_ERROR
-          'Trying to verify NSEC RRset with RRSIG {sig} gave error \'{error}\'.', @_;
+'Trying to verify NSEC RRset with RRSIG {keytag} gave error \'{error}\'.',
+          @_;
```
Comment (matsduf) [inline lib/Zonemaster/Engine/Test/DNSSEC.pm:816]: Indentation looks wrong.

File: lib/Zonemaster/Engine/Test/DNSSEC.pm [hunk 3]
```diff
@@ -842,11 +858,12 @@ Readonly my %TAG_DESCRIPTIONS => (
     },
     SOA_SIGNATURE_NOT_OK => sub {
         __x    # DNSSEC:SOA_SIGNATURE_NOT_OK
-          'Trying to verify SOA RRset with signature {signature} gave error \'{error}\'.', @_;
+'Trying to verify SOA RRset with signature {keytag} gave error \'{error}\'.',
+          @_;
```
Comment (matsduf) [inline lib/Zonemaster/Engine/Test/DNSSEC.pm:862]: Indentation.

**Review by matsduf — CHANGES_REQUESTED** [review]
There are several data types that are actually a non-negative, decimal integer (e.g. keytag and SOA serial). One possibility is to have a common data type to have fewer data types. Another (the path you have taken) is to have specific data type for each use.

This is a good work, and when more is completed we will see if this path will give too many data types.

File: docs/logentry_args.md [hunk 4]
```diff
@@ -4,12 +4,19 @@
 
 | Argument    | Type of value        | Description and formatting                                  |
 |-------------|----------------------|-------------------------------------------------------------|
+| algo_descr  | Text                 | The human readable description of a DNSKEY algorithm.       |
+| algo_mnemo  | Text                 | The mnemonic of a DNSKEY algorithm.                         |
```
[the remark this answers is not in the document]
Comment (mattias-p) [reply docs/logentry_args.md:8]: Is it good enough if I change it to "DNSSEC algorithm" and link to https://www.iana.org/assignments/dns-sec-alg-numbers/dns-sec-alg-numbers.xhtml?

File: docs/logentry_args.md [hunk 5]
```diff
@@ -4,12 +4,19 @@
...
+| algo_num    | Number               | The numeric value for a DNSKEY algorithm.                   |
+| keytag      | DS/DNSKEY/RRSIG keytag | A keytag for a DS, DNSKEY or RRSIG record.                |
+| module      | A Zonemaster test module, or `all` | The name of a Zonemaster test module.         |
```
[the remark this answers is not in the document]
Comment (mattias-p) [reply docs/logentry_args.md:11]: I'd prefer not to consider argument value syntax as part of this series of pull requests, keeping to a narrow reading #713, limited to names and semantics. Syntax is also important, of course, but I'd like to deal with that in a separate issue. Would you be okay with that?

File: docs/logentry_args.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (matsduf) [reply docs/logentry_args.md]: For now, yes. Later I think we should define it in the test case specification and then use the same string in the implementation.

File: docs/logentry_args.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (matsduf) [reply docs/logentry_args.md:11]: I do not fully understand, but I think it is acceptable that this update does not create the perfect solution and that we come back later with improvements.

**Review by matsduf — APPROVED** [review]

**Pull request merged**